crates/runtime/src/objectstore/ftp.rs

```diff
@@ -122,6 +123,13 @@ fn pipe_stream(
         let mut total = 0;
         let mut buf = vec![0; 4096];
 
+        client
+            .transfer_type(FileType::Binary)
```

Comment:
should this be configurable in params?

Reply:
No, we need to use binary to be able to resume_transfer